src/protagonist/Orchestrator/Features/Images/ImageServer/InfoJson2Constructor.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Model.Assets;
+using IIIF;
+using IIIF.ImageApi.V2;
+using Microsoft.Extensions.Logging;
+using Orchestrator.Assets;
+using Orchestrator.Infrastructure.IIIF;
+using IIIFAuth2 = IIIF.Auth.V2;
+using Version = IIIF.ImageApi.Version;
+
+namespace Orchestrator.Features.Images.ImageServer;
+
+/// <summary>
+/// Service responsible for orchestrating image, calling IImageServerClient to get info.json and update with
+/// required information that image-server will be unaware of (e.g. Auth, Id).
+/// </summary>
+public class InfoJson2Constructor : InfoJsonConstructorTemplate<ImageService2>
```

Comment:
is it worth adding to this class description what the difference is between this, `InfoJsonConstructor` and `InfoJson3Constructor` - assuming the difference is with the `AuthBuilder`

Reply:
Done, this was copy/paste from original `InfoJsonConstructor`